docs/02-guides/extend-vite-configuration/tailwind.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Tailwind
+description:
+  This guide explains how to setup Tailwind in your Front-Commerce application
+  thanks to Vite.
+---
+
+<p>{frontMatter.description}</p>
+
+:::tip
+
+You can check the `example-extension/tailwind` folder from the skeleton to check
+out it's usage.
+
+:::
+
+## Install Tailwind
+
+```sh
+pnpm add -D tailwindcss
+```
+
+## Generate Tailwind configuration
+
+```sh
+pnpm dlx tailwindcss init -p --ts
+```
+
+## Update your tailwind config
+
+Update your `tailwind.config.ts` to include files from `app/theme`:
+
+```diff
+import type { Config } from 'tailwindcss'
+
+export default {
+- content: []
++ content: ["./app/**/*.{ts,tsx,js,jsx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+} satisfies Config
+```
+
+If you have components in some of your extensions, you need to add the related
+extensions' paths to the `content` directive.
+
+## Setup PostCSS
+
+The `tailwindcss init` command should have generated a `postcss.config.js` file,
+you can remove this file and add the `tailwind` configuration to the
+`postcss.config.mjs` file instead:
+
+```diff
+export default {
+  plugins: {
+    autoprefixer: {},
++    tailwindcss: {},
+  },
+};
+```
+
+## Inject Tailwind stylesheet
+
+Create a `tailwind.css` file in the `app` folder with the following content:
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Then, import this file in your `root.tsx` like so:
+
+```diff
+...
++ import "./tailwind.css";
+...
+```
+
+You can now start your project and tailwind will be used for all your styles.
+
+# Usage
+
+You can test it by loading this module into your app by inject it in your
+`front-commerce.config.ts` file:
+
+```diff
+...
++import tailwind from "./example-extensions/tailwind";
+
+export default defineConfig({
+  extensions: [
+    themeChocolatine(),
++    tailwind()
+  ],
+...
+```
+
+And update the `tailwind.config.js` with this:
+
+```diff
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./app/theme/**/*.{ts,tsx,js,jsx}",
++    "./example-extensions/tailwind/theme/**/*.{ts,tsx,js,jsx}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+Now you can head to
+[`http://localhost:4000/tailwind`](http://localhost:4000/tailwind) and check
+that Tailwind is working.
+
+## More infos
+
+If you want more infos about Tailwind setup with Remix, please check
+[this guide](https://tailwindcss.com/docs/guides/remix).
```

Comment:
I don't think this deserves a title, rather at the start of this docs add a references section with link to tailwind and remix

```suggestion
## References

- https://tailwindui.com/documentation
- https://tailwindcss.com/docs/guides/remix
```